### DISCUSSION TITLE
How do I recognize impending icing over Heishamon (L model)

### DISCUSSION THREAD

**Discussion by AlexanderSch90** [Egyras/HeishaMon#572] [Q&A] [answered]
Hello everyone,

I operate two Panasonic L-model air-to-water heat pumps in cascade mode without a cascade manager. It is controlled by two Heishamon in combination with my home automation system.

I would like to find out how I can recognize that the first heat pump is about to freeze up so that I can switch it off and switch on the second heat pump at the same time. I am hoping for no thawing during the wet and cold period. I am looking for specific tips or experience on how to better assess icing.

Does anyone have similar experiences or helpful tips? Many thanks

**Comment by MiG-41** [comment, marked as answer]
In service manual for each type there is a "deice chart" - showing at with Outdoor temperature what Outdoor pipe temperature should be.
This is example for M model: 

![image](https://github.com/user-attachments/assets/7f1dede1-83c2-44cd-9638-2779991f1937)


For your can be a little bit different temps.
So you have to implement such graph in your automation , and stop HP 2 minutes after Outdoor pipe temperature drups to this particular value ( depends on Outdoor temperature).
EDIT: You have to implement also this 35minutes.... When you start such freezed HP , it will also run 35min with COP=1 or even below , becouse it will not do the deicing before that....

**Reply by AlexanderSch90** [reply]
I found the same graphic on my model. Thank you for pointing me to the right place. My idea is that during the time HP2 is running, HP1 is regenerated by the outside air. And the other way around. This is because one HP is sufficient for our house up to about -2 degrees Celsius.

**Comment by rvdgaag** [comment]
This is probably never going to work, de-icing needs some reverse heating of your condensor, not going to happen by itself by the outside temperature, maybe when its 10 degrees above zero but not at the typical outside temperatures where the condensors builds up ice.

**Reply by geduxas** [reply]
It's not about outside temperature, humidity is main factor for icing.

And yes, for defrost, pump switching in reverse mode, so it's start to cool inside..

**Comment by AlexanderSch90** [comment]
Correct, it does not work. So the graphic fits, but the idea behind it doesn't work. What I have done now: If one heat pump would actually be enough, but the environmental conditions cause frequent defrosting, then the second heat pump will start up anyway and both will run at the minimum setting. According to my tests, this increases the defrosting time considerably (by a factor of more than 2).

**Reply by geduxas** [reply]
You could force defrost manually by monitoring it's temperature. And if you could measure outdoor humidity, you could calculate and predict icings..

**Comment by gvzdus** [comment]
Very curious about your further findings!
I have the same setup with 2 Aquarea LTs (but with cascade manager - even if disabled currently - see my self-introduction in the forum).
Yes, my impression is also: "2 HP running => less defrosting in total".
However, I do have the problem that defrosting of HP1 triggers a H62 error on the other HP, and vice versa. At least with a 30-70% chance.
Btw: geduxas: That is one reason why I would really *love* to have a reset option as Set-command:
Imagine my panic that I am on the leave, the error happens, the circulation pump ceases work because of the error, and the building is getting < 0°!

**Reply by geduxas** [reply]
H62, is water flow abnormal. It could mean that your installation is not properly done, do you have buffer tank? Or hydraulic separator between pump's? 

For reset error is still possible, you can send custom command.. as the @MiG-41 has already told..

**Reply by gvzdus** [reply]
Yes, it is linked with the peak water flow. Discussed with the company: He increased the pump settings and the error was - at least - not showing up that fast.
However, I also asked (complained) about the buffer scheme. How they set it up initially: As a seperator: One ~200l buffer with 4 pipes: HP in, HP out, heating circle in (fwd) & out (return). Result: With e.g. 35° heating circle in & 30° heating circle return & 20l/min, the heatpump had to deliver 40° (with 35°, because HPin = heating circle in) at 20l/min or even more at lower l/min.
Now, I have a manual valve that I can close and typically run with the buffer valve closed: Heating circle return = HP in.

**Reply by PGnl** [reply]
Could it be that extreme flow of a defrost causes too much backpressure on the other heatpump giving fault?  I mean that the >30l/min of one in defrost starts too push the other (almost) in reverse as it is running at low speeds/power. 

is it only for one if the two? If so, it could be that piping is made wrong.

**Reply by gvzdus** [reply]
Nope, it is - with high chance - always for both (HP1 defreeze: HP2 H62, HP2 defreeze: HP1).
There *are* direction valves, but it really seems to be a matter of back pressure.

**Reply by geduxas** [reply]
> Yes, it is linked with the peak water flow. Discussed with the company: He increased the pump settings and the error was - at least - not showing up that fast. However, I also asked (complained) about the buffer scheme. How they set it up initially: As a seperator: One ~200l buffer with 4 pipes: HP in, HP out, heating circle in (fwd) & out (return). Result: With e.g. 35° heating circle in & 30° heating circle return & 20l/min, the heatpump had to deliver 40° (with 35°, because HPin = heating circle in) at 20l/min or even more at lower l/min. Now, I have a manual valve that I can close and typically run with the buffer valve closed: Heating circle return = HP in.

If i understand correctly, in your setup, pipes are coming to one and after it will go to buffer? It's indeed wrong.. i think it should be seperate inputs for pumps.. or at last use manifold with hydraulic seperator and after that input 2 pipes in buffer tank..

**Reply by gvzdus** [reply]
For 3 days, it is working fine now:
Whenever "Defrosting_State" goes to 1, I set both HPs to Max_Pump_Duty 125. Once "Defrosting_State" goes to 0, I put the value back to a senseful setting (87 for dual operation, 105 for single operation).
Pretty happy now about the stability!
